libgeopmd/src/DrmSysfsDriver.hpp

```diff
@@ -37,8 +37,8 @@ namespace geopm
             static std::unique_ptr<IOGroup> make_plugin(void);
         private:
             const std::map<std::string, SysfsDriver::properties_s> M_PROPERTIES;
+            const std::map<std::pair<geopm_domain_e, int>, std::string> M_DRM_HWMON_DIR_BY_GEOPM_DOMAIN;
```

Comment:
A comment describing the key for this map might  be handy.  It is made more obvious by using the geopm_domain_e type rather than an int, as is commonly done in the code base with our enums (i.e. i like the style change).  But, I still was confused about the pair that was the key without looking into the implementation a bit.

Reply:
Adding comment: `Map of (GEOPM signal domain, GEOPM signal index) pairs to hwmon sysfs directory paths`